docs/user-manual/modules/ROOT/pages/rest-dsl.adoc

```diff
@@ -317,6 +317,13 @@ to specify a custom content-type if the message body should not attempt to be
 marshalled using the binding. For example if the message body is a
 custom binary payload etc.
 
+When automatic binding from POJO to JSon/JAXB takes place the existing `content-type` header will by default be replaced with either `application/json` or `application/xml`. To disable the default behavior and be able to produce JSon/JAXB responses with custom `content-type` headers (e.g. `application/user.v2+json`) you configure this in Java DSL as shown below:
```

Comment:
Maybe "you should configure" instead of "you configure"

Reply:
I tried to be consistent with how the text was written for the example below my changes (i.e. "To enable binding you configure this in Java DSL as shown below:") but I think it looks better with "should". Should I change it or keep it as it is?

Reply:
I think it's fine as is, we could always change later. Thanks.